docs/src/details/integration/building/cmake.rst

```diff
@@ -70,11 +77,27 @@ If you use cmake to install lvgl 3 folders will be created.
 
 Build with Command line
 -----------------------
+You can also build your project using the command line. Run the following commands to use the presets:
+
+.. code-block:: bash
+
+    cmake --preset windows-base
+    cmake --build --preset windows-base_dbg
+    ctest --preset windows-base_dbg
+
 
-You can also build your project using the command line. Run the following commands
+To not use preset, use these commands:
 
-- ``cmake --preset windows-base``
-- ``cmake --build --preset windows-base_dbg``
-- ``ctest --preset windows-base_dbg``
+.. code-block:: bash
```

Comment:
I suggest starting with these instead of presets and these commands are more generic.